utils/resb/src/binary.rs

```diff
@@ -152,7 +112,8 @@ primitive_enum!(
     u8,
     /// The endianness used to write a resource bundle.
     #[derive(Clone, Copy, Debug, PartialEq)]
-    enum Endianness {
+    #[allow(clippy::exhaustive_enums, missing_docs)]
```

Comment:
I added `missing_docs` because otherwise it complains about a lack of documentation for the variants. In theory, I could put something there, but... what?